unlock-protocol.com/blog/digital-cash.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Digital Cash. Big Deal.
+subTitle: Wallets and their superpowers for users.
+authorName: Sascha Mombartz
+publishDate: June 24, 2019
+description: A quick exploration into blockchain wallets, their applications and why they're so powerful.
+image: /static/images/blog/digital-cash/digitalcash.jpg
+---
+
+The wallet. It’s your interface into the blockchain. It lets you transact with the blockchain and in Ethereum’s case also with its smart contracts. You could think of it as an account but its more of an identifier with the crucial difference that you are in possession of it. It’s not stored on a server but lives with you, on your computer, your phone or hardware wallet.
```

Comment:
I would remove "but its more of an identifier" because it is more than an identifier (which would probably just be public?).

An account is actually the combination of a public key and a private key. Only the former is an "identifier" :)

Reply:
making the image smaller also doesnt center it and looks not so good on retina...